nva-datamodel-java/src/test/java/no/unit/nva/model/util/PublicationGenerator.java

```diff
@@ -101,6 +115,50 @@ public static Publication generatePublication(String type) throws InvalidIssnExc
                                                                       InvalidUnconfirmedSeriesException {
         Reference reference;
         switch (type) {
+            case "ArtisticDesignClothingDesign":
```

Comment:
Maybe in the future we should consider something like this: https://stackoverflow.com/questions/347248/how-can-i-get-a-list-of-all-the-implementations-of-an-interface-programmatically ?